uaclient/security.py

```diff
@@ -943,10 +946,12 @@ def prompt_for_affected_packages(
                     )
                 )
             )
+            return False
```

Comment:
I wasn't a part of the initial discussion about this feature, so maybe this was already discussed: I think we should double check that we want to return False in this case.

My first intuition would be that this would be considered success: `ua fix` succesfully installed the packages to fix the vulnerability. The only difference is that the user also needs to restart before the new binaries/libs are loaded (or whatever reason the restart is necessary). 

IOW: Technically the system is not in a non-vulnerable state because the reboot is required, but ua fix did everything that it can to solve the problem successfully, so I would expect ua to return success.

I'm open to opinions otherwise ofc, and maybe we need to bring up in the foundations meeting tomorrow

Reply:
Yes, I think this is a good point to bring for the foundations meeting. My rationale behind that is exactly what you pointed out, that the system is still in a non-vulnerable state. Another issue is that if we have a package that needs reboot prior to running `ua fix` we would say that a reboot is needed and return False here in a situation that maybe the system is non-vulnerable.

So yes, let bring it up on the meeting

Reply:
Result of the meeting: Let's return `2` in this case, since the system is not definitely in a nonvulnerable state, but we want to differentiate it from a normal failure (where we fail to install the packages for some reason)